libs/accounts/src/lib/accounts-table.spec.tsx

```diff
@@ -71,6 +93,31 @@ describe('AccountsTable', () => {
     ).toEqual(expectedHeaders);
   });
 
+  it('should render deposit button', async () => {
+    const { container } = render(
+      <AccountTable
+        rowData={zeroBalanceRowData}
+        onClickAsset={() => null}
+        isReadOnly={false}
+        onClickDeposit={onClickDepositMock}
+        pinnedAsset={{
+          decimals: 5,
+          id: '5cfa87844724df6069b94e4c8a6f03af21907d7bc251593d08e4251043ee9f7c',
+          symbol: 'tBTC',
+          name: 'tBTC',
+        }}
+      />
+    );
+    const depositButton = container.querySelector('[data-testid="deposit"');
+    expect(depositButton).toBeVisible();
+    if (depositButton) {
+      fireEvent.click(depositButton);
```

Comment:
Please use userEvent unless theres a good reason not to
```suggestion
    await userEvent.click(depositButton);
```